tests/tasks/create_test_interfaces.yml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Maybe call this file `create_test_interfaces_with_dhcp_server.yml` to indicate that it will setup dhcp, too.

Reply:
Another thought, since we need to use dhcp with hostapd as well, can we maybe re-use this dhcp setup there, to avoid duplication?